databroker/broker.py

```diff
@@ -233,6 +234,25 @@ def __getitem__(self, key):
         """Do-What-I-Mean slicing"""
         return search(key, self.mds)
 
+    def __getattr__(self, key):
+        try:
+            query = self.aliases[key]
+        except KeyError:
+            raise AttributeError(key)
+        if callable(query):
+            query = query()
+        return self(**query)
+
+    def alias(self, key, **query):
```

Comment:
Too late to bikeshed, darn. `alias` being ambiguously as a verb/noun and there being no docstring, I was confused to where this was going...